src/covid19sim/simulator.py

```diff
@@ -1177,7 +1177,7 @@ def at(self, location, city, duration):
         self.wear_mask()
 
         self.start_time   = self.env.now
-        self.leaving_time = self.start_time + duration*SECONDS_PER_MINUTE*TICK_MINUTE
+        self.leaving_time = self.start_time + duration*SECONDS_PER_MINUTE
```

Comment:
Is it that we are moving towards all time-units for yielding to be in SECONDS? It is good change!

Reply:
After #147 was merged, all time-units for yielding became seconds.
After this PR is merged, no code anywhere will use ticks and `TICK_MINUTE` is gone.
However, most remaining magic time constants in the code continue to be in units of either days, hours or minutes, and most `duration`s for an activity are in minutes. For that reason, #147 causes every function that `yield`s to maintain backward compatibility in its external interface (usually minutes), while internally translating to seconds.

In upcoming patches I will hunt down all uses of time and scale them to seconds.